src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -295,7 +297,7 @@ internal static class ConstraintsHelper
                 return null;
             }
 
-            var bounds = new TypeParameterBounds(constraintTypes, interfaces, effectiveBaseClass, deducedBaseType);
+            var bounds = new TypeParameterBounds(constraintTypes, interfaces, effectiveBaseClass, deducedBaseType, ignoresNullableContext);
```

Comment:
>ignoresNullableContext [](start = 115, length = 22)

It doesn't look like this value actually affects other content in the TypeParameterBounds instance. Why not always use ```false``` and and getting rid of the parameter.